applications/pusher/src/modules/pm_apple.erl

```diff
@@ -98,3 +111,8 @@ maybe_load_apns(App, ETS, CertBin) ->
             lager:error("Error loading apns ~p", [Reason]),
             'undefined'
     end.
+
+-spec apns_topic(kz_json:object()) -> binary().
+apns_topic(JObj) ->
+    TokenApp = kz_json:get_ne_binary_value(<<"Token-App">>, JObj),
+    re:replace(TokenApp, <<"\.(?:dev|prod)$">>, <<>>, [{'return', 'binary'}]).
```

Comment:
You need to escape the backslash for it to have value in the eyes of `re`, as `"\." = "."`.